<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MergeFileSelection</class>
 <widget class="QWizardPage" name="MergeFileSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Select Files</string>
  </property>
  <property name="subTitle">
   <string>Choose the files to be merged. If you do not know what Big Endian means, let the column be in its default state.</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item alignment="Qt::AlignLeft">
      <widget class="QPushButton" name="addFilesTextFile">
       <property name="toolTip">
        <string>from a text file that contains file locations in separate lines.</string>
       </property>
       <property name="text">
        <string comment="from a text file that contains file locations in separate lines.">Add from Text File</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignLeft">
      <widget class="QPushButton" name="addFileManual">
       <property name="text">
        <string>Add Files</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QCheckBox" name="showLocationCheck">
       <property name="text">
        <string>Show Locations</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="filesTable">
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="showGrid">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>File</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Location</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Big-Endian</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="saveFileButton">
       <property name="text">
        <string>Save File as</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="saveFileEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
